Sources/Cloud/Commands/Database/DatabaseCreate.swift

```diff
@@ -0,0 +1,229 @@
+            "in a Git controlled folder with a remote matching",
+            "the application's hosting Git URL."
+            ])
+    ]
+    
+    public let help: [String] = [
+        "Create a new database server"
+    ]
+    
+    public let console: ConsoleProtocol
+    public let cloudFactory: CloudAPIFactory
+    
+    public init(_ console: ConsoleProtocol, _ cloudFactory: CloudAPIFactory) {
+        self.console = console
+        self.cloudFactory = cloudFactory
+    }
+    
+    public func run(arguments: [String]) throws {
+        console.info("Create new database")
+        
+        console.info("")
+        console.warning("Be aware, this feature is still in beta!, use with caution")
+        console.info("")
+        
+        let token = try Token.global(with: console)
+        let user = try adminApi.user.get(with: token)
+        
+        let app = try console.application(for: arguments, using: cloudFactory)
+        
+        let environments = try applicationApi.environments.all(for: app, with: token)
+        
+        var envArray: [String] = []
+        
+        console.info("")
+        console.info("This creates a database server for the application: \(app)")
+        console.info("It will also setup a database on the server for each of the environments:")
+        
+        try environments.forEach { val in
+            console.info(" - \(val)")
+            envArray.append("\(val.id ?? "")")
+        }
+        console.info("")
+        
+        let name = console.ask("What name for this database server? [a-zA-Z0-9]")
+        console.detail("name", name)
+        let size = try self.size(for: arguments)
+        let type = try self.type(for: arguments)
+        var version: String = ""
+        // TODO: Switch to API based versions, for more flexibility
+        switch type {
+        case .mysql:
+            version = try self.MySQLVersions(for: arguments)
+        case .postgresql:
+            version = try self.PostgresVersions(for: arguments)
+        case .mongodb:
+            version = try self.MongoDBVersions(for: arguments)
+        }
+        let access = token.access        
+        
+        try CloudRedis.createDBServer(
+            console: console,
+            cloudFactory: cloudFactory,
+            application: app.repoName,
+            name: name,
+            size: "\(size)",
+            engine: "\(type)",
+            engineVersion: version,
+            environments: envArray,
+            userToken: access,
+            email: user.email
+        )
+    }
+    
+    private func size(for arguments: [String]) throws -> Size {
+        let size: Size
+        
+        console.pushEphemeral()
+        
+        size = try console.giveChoice(
+            title: "Which size?",
+            in: Size.all
+        ) { type in
+            switch type {
+            case .free:
+                return "(Dev) Free $0/month - (256mb memory - 10,000 records - 20 connections - no backups)"
```

Comment:
1: what does `(Dev)` mean?
2: slashes could be nice `(256MB memory / 10K Rows / 20 Connections)`
we probably don't need to say "no backups", just saying "yes backups" for the plans that have them would work

Reply:
Dev relates to the different plans, dev meaning free/hobby .. Later they will be changed to questions, so you select:

- Dev
- Standard
- Professional

And then the specific plan after :) .. Just to stress out now it's not for production in any way :)